build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
     id "org.jetbrains.kotlin.jvm" version "1.3.61"
     id "edu.wpi.first.GradleRIO" version "2020.1.2"
-//    id "org.jlleitschuh.gradle.ktlint" version "9.1.1"
```

Comment:
This actually will not turn off the ktlint checking because the CI still calls `gradlew ktlintCheck`. But because this task doesn't exist due to the commented out plugin, it fails 😢 Instead we should probably just deal with ktlint for this pr